crates/solvers/src/domain/auction.rs

```diff
@@ -8,13 +8,31 @@ use {
 #[derive(Debug)]
 pub struct Auction {
     pub id: Option<Id>,
-    pub tokens: HashMap<eth::TokenAddress, Token>,
+    pub tokens: Tokens,
     pub orders: Vec<order::Order>,
     pub liquidity: Vec<liquidity::Liquidity>,
     pub gas_price: GasPrice,
     pub deadline: chrono::DateTime<chrono::Utc>,
 }
 
+/// Information about tokens used in the auction.
+#[derive(Debug)]
+pub struct Tokens(pub HashMap<eth::TokenAddress, Token>);
```

Comment:
Nice, I think this is more coherent than the existing `dex::slippage::Prices` and the previously proposed passing in `auction::Auction`. Improvement to the `domain`!